src/daemon/daemon.cpp

```diff
@@ -2438,9 +2438,9 @@ try
 
     auto key = request->key();
     auto val = request->val();
-    auto bridge_name = get_bridged_interface_name();
+    auto bridge_name = MP_SETTINGS.get(mp::bridged_interface_key).toStdString();
```

Comment:
IIUC, you should only need to read this if we are indeed dealing with authorized bridges. For any other set requests, this adds a pointless disk read. Therefore, can we read this only if the request is authorized? That would filter most cases.

Reply:
I understand you need the `bridge_name` var in this scope to later remove it (and you will still need it with the scope guard). But you can initialize it as empty by default, and only fetch the actual value when necessary.

Reply:
Right, done.